Library/Homebrew/cmd/desc.rb

```diff
@@ -0,0 +1,47 @@
+require 'descriptions'
+require 'cmd/search'
+
+module Homebrew
+  def desc
+    options = ARGV.options_only
+    cache_op = false
```

Comment:
I was trying to cater to the user who wanted to delete the cache before re-creating it with a search, and do so with a single command, but you’re right. The “pick one” logic is much cleaner.